packages/complaints/test/constants/test_constants.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:complaints/complaints.dart';
+import 'package:complaints/models/entities/complaint_types.dart';
+import 'package:digit_data_model/data_model.dart';
+import 'package:digit_data_model/models/entities/pgr_application_status.dart';
+
+class ComplaintsRegistrationConstants{
+  static final ComplaintsDetailsModel complaintsDetailsModel = ComplaintsDetailsModel(
+      dateOfComplaint: DateTime(2024),
+      administrativeArea: "abcd",
```

Comment:
Update constant values with a dummy data. Make sure the dummy data is of same format as the original values in the application.